model/users_model.js

```diff
@@ -62,7 +62,18 @@ module.exports.findUser = async (username) => {
   }
 
   try {
-    return Promise.all([userPromise, postsPromise, followersPromise, followingPromise])
+    return await Promise.all([userPromise, postsPromise, followersPromise, followingPromise])
```

Comment:
Whats up with this?

Reply:
Found a bug where if a user is not found, it'll still return 200 because no Promise rejection is thrown till it reaches the controller. This is to make sure the rejection is thrown in the model.